dd-java-agent/src/main/java/datadog/trace/bootstrap/AgentJar.java

```diff
@@ -30,6 +30,9 @@ public static void main(final String[] args) {
           case "uploadCrash":
             uploadCrash(args);
             break;
+          case "scandependencies":
```

Comment:
```suggestion
          case "scanDependencies":
```

To maintain consistency with case style of other commands.